src/MeshKernelNET/Api/MeshKernelApi.cs

```diff
@@ -96,6 +96,15 @@ public int ContactsGetData(int meshKernelId, out DisposableContacts disposableCo
             return MeshKernelDll.CurvilinearComputeOrthogonalGridFromSplines(meshKernelId, ref geometryListNative,
                                                                              ref curvilinearParametersNative, ref splinesToCurvilinearParametersNative);
         }
+        
+        public int CurvilinearComputeSmoothness(int meshKernelId, int direction, ref double[] smoothness)

```

Comment:
When `enum CurviliearDirection` is used, you will need use  `Convert.ToInt32(direction)` in the second param of `MeshKernelDll.CurvilinearComputeSmoothness`.